### ISSUE TITLE
Warning from numpy1.9.0 in ESMP_API.py and avariable.py

### ISSUE THREAD

**Issue by chaosphere2112** [CDAT/cdms#62]
[durack1@oceanonly 140115_PeterGleckler]$ /usr/local/uvcdat/2014-07-21/bin/python
Python 2.7.7 (default, Jul 21 2014, 15:49:32)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.

> > > import numpy
> > > numpy.version.version
> > > '1.9.0b1'

```
/usr/local/uvcdat/2014-07-21/lib/python2.7/site-packages/ESMP/src/ESMP_API.py:1241: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  if (srcMaskValues != None):
```

and same file another line:

```
/usr/local/uvcdat/2014-08-22/lib/python2.7/site-packages/ESMP/src/ESMP_API.py:1248: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  if (dstMaskValues != None):
```

Migrated from: https://github.com/UV-CDAT/uvcdat/issues/576

**Comment by zshaheen** [comment]
@dnadeau4 This is still an issue. Not sure what the solution is or if anyone in this team can do something about this.

**Comment by durack1** [comment]
@zshaheen theoretically #57 (UV-CDAT/uvcdat#1151) should solve this, the current `cdms2` `ESMP` release is 6.3.0 whereas [7.0.0 has been out for a while](https://www.earthsystemcog.org/projects/esmpy/releases)

**Comment by zshaheen** [comment]
@durack1 Thanks for the heads up. Glad to know that there is a possible solution. Hopefully someone will bother upgrading soon.

**Comment by doutriaux1** [comment]
@zshaheen it's on @dnadeau4 list but if you have cycles and it really bothers you I'm sure he won't mind :wink:

**Comment by dnadeau4** [comment]
This warning has been fix with ESMF 7.0.0
